src/assets/style/main.scss

```diff
@@ -1,6 +1,20 @@
 $color-green:#008000;
 
-.text
-{
+.text {
   color: $color-green;
-}
\ No newline at end of file
+}
+
+.modal {
+  text-align: right !important;
+}
+
+.ui.active.transition.visible.dimmer,
+.ui.page.modals.dimmer.transition.visible.active {
```

Comment:
which class is this and wouldn't have been easier to create a new reference class ?

Reply:
Both classes are inaccessible directly, they are generated by semantic UI React components, that is why I chose to pick the whole class as is to effect the change that was needed.